applications/CompressiblePotentialFlowApplication/tests/naca0012_small_perturbation_compressible_test/naca0012_small_perturbation_compressible_parameters.json

```
@@ -0,0 +1,104 @@
+{
+    "problem_data"                     : {
+        "problem_name"    : "naca0012",
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 1
+    },
+    "solver_settings"                  : {
+        "solver_type"            : "potential_flow",
+        "domain_size"     : 2,
+        "model_part_name" : "MainModelPart",
+        "model_import_settings"  : {
+            "input_type"     : "mdpa",
+            "input_filename" : "naca0012_small"
+        },
+        "formulation": {
+                    "element_type":"perturbation_compressible"
+        },
+        "maximum_iterations"     : 30,
+        "echo_level"             : 0,
+        "relative_tolerance": 1e-12,
+        "absolute_tolerance": 1e-12,
+        "linear_solver_settings"  : {
+                "solver_type"             : "ExternalSolversApplication.super_lu",
```

```suggestion
                "solver_type"             : "EigenSolversApplication.sparse_lu",
```
`ExternalSolversApplication` is going to be deprecated soon.